### ISSUE TITLE
Pathwhiz Nodes 

### ISSUE THREAD

**Issue by sstemann** [NCATSTranslator/Feedback#541]
I'm guessing these are valid but I'm not sure if it makes sense in these analysis graphs? 

![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/4e838b69-0f43-4560-8794-f3448b6c7740)

Maybe it makes sense to someone else but would i read it like this:

Hydrogen Donor > is input of > "Reduced acceptor + Vitamin A → Acceptor + All-trans-13,14-dihydroretinol" > has input  > Retinol > treats "Familial Pityriasis Rubra Pilaris"

![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/fae6ffec-2fd6-48f5-9201-bb8144651d3d)

Steps to recreate:
1. UI Prod > Login
2. Drug > Disease "Familial Pityriasis Rubra Pilaris": https://ui.transltr.io/main/results?l=Familial%20Pityriasis%20Rubra%20Pilaris&i=MONDO:0008251&t=0&q=17e013e2-9da0-4a60-b650-3a21eb8772ba
3. Text search Pathwhiz
4. Expand a path
5. Get a Pathwhiz ID from Translator
6. Get a Pathwhiz account, access the Pathwhiz id:  https://pathbank.org/lims#/reactions/110448

Looks like the 9 results with pathwhiz are from ARAGORN, BTE has 1 with this text bc it was merged with Water, but BTE does not return pathwhiz nodes

related to #540

**Comment by gglusman** [comment]
This seems to me almost the same issue as #468. The 'result' is different but the rest of the path is the same, with the same underlying failure - incorrect assumption of transitiveness.

**Comment by sstemann** [comment]
@gglusman I wanted to raise this particular issue about this node type and its intent to the ARAGORN team, I know in the past they have used Foo DB nodes and made adjustments independently.

**Comment by sstemann** [comment]
@uhbrar just checking if this is intentional to have this pathwhiz reaction nodes and if so, is there a name that should be displayed? or a link out that can be used?

maybe @dnsmith124 can advise on the display of these node types?

2/15/2024 Test UI:  https://ui.test.transltr.io/main/results?l=Familial%20Pityriasis%20Rubra%20Pilaris&i=MONDO:0008251&t=0&r=0&q=4ee5521b-e47b-464b-93f1-2b5caa57bbf4

![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/fe965ecd-7f4f-4c99-82d8-8c22ce8bdcbe)

https://arax.ncats.io/beta/?r=80fcf2eb-343d-483d-9768-7db70ca9a49c

![image](https://github.com/NCATSTranslator/Feedback/assets/38321826/f1bdfc73-5189-4c93-b023-5a7b1adf188d)

https://pathbank.org/pathwhiz/users/sign_in#

**Comment by sstemann** [comment]
@uhbrar @dnsmith124 @MarkDWilliams now these say "nameunknown" in the UI and the ARAX GUI.  it doesnt seem like a good look, is this something that can get fixed?

<img width="930" alt="image" src="https://github.com/NCATSTranslator/Feedback/assets/38321826/c7c01914-1e67-4456-8bfa-20792074b27d">

<img width="869" alt="image" src="https://github.com/NCATSTranslator/Feedback/assets/38321826/0b4fb31b-5c58-4096-a039-875832cfb66b">

https://ui.test.transltr.io/main/results?l=Familial%20Pityriasis%20Rubra%20Pilaris&i=MONDO:0008251&t=0&r=0&q=91b5aece-d265-4934-ae9a-e642917de91a

**Comment by gaurav** [comment]
I've opened an issue for adding PathWhiz identifiers to NodeNorm, but it'll probably take a couple of months before we can get to it: https://github.com/TranslatorSRI/Babel/issues/254

**Comment by sstemann** [comment]
by couple of months are you thinking sometime in the fall?

**Comment by sstemann** [comment]
duplicate of #465
